Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -772,6 +821,14 @@ public void Deconstruct()
             furniture.Jobs.CancelAll();
         }
 
+        if (deconstructInventory != null)
+        {
+            foreach (Inventory inv in deconstructInventory)
+            {
+                World.Current.inventoryManager.PlaceInventory(Tile, inv);
```

Comment:
I'm not really sure if it would work either, but I'm not sure what the solution is to it. It almost seems like the InventoryManager should just pick a neighbor tile if the current one is occupied, but I'm not sure if I want to do something like that in this PR. I could look in to it if you think it's an issue, however.